index.html

```diff
@@ -59,5 +59,9 @@ <h1 class="mb-4">Enter Date</h1>
 
     </div>
 
+    <div class="footer">
+        <p>Check out the project on <a href="https://github.com/gratten/blockstamp" target="_blank">GitHub</a>.</p>
```

Comment:
My suggestion is to rephrase it slightly, something to the effect of "Check out the code for the project on..."